server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimGroupMembershipManager.java

```diff
@@ -118,10 +118,9 @@ private Set<ScimGroup> getDefaultUserGroups(String zoneId) {
             return emptySet();
         }
         IdentityZone currentZone = IdentityZoneHolder.get();
-        List<String> zoneDefaultGroups = currentZone.getConfig().getUserConfig().getDefaultGroups();
-        if (!zoneId.equals(currentZone.getId())) {
-            zoneDefaultGroups = zoneProvisioning.retrieve(zoneId).getConfig().getUserConfig().getDefaultGroups();
-        }
+        List<String> zoneDefaultGroups = (zoneId.equals(currentZone.getId())) ?
```

Comment:
> Task :cloudfoundry-identity-uaa:gemInstallBundle
Ignoring eventmachine-1.2.7 because its extensions are not built. Try: gem pristine eventmachine --version 1.2.7
Ignoring http_parser.rb-0.6.0 because its extensions are not built. Try: gem pristine http_parser.rb --version 0.6.0
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.6.0 directory.

Reply:
What might be the reason for this error?

Reply:
this is a local issue, in case of problems install ruby with rvm https://rvm.io/ then you dont need root

Reply:
use intellij and run docs tests manually, but only LogoutInfoEndpointDocs, maybe you have to change an assert there

LogoutInfoEndpointDocs > logout() FAILED
    java.lang.AssertionError at LogoutInfoEndpointDocs.java:44

Reply:
@strehle now the voter is green